templates/404.html

```diff
@@ -4,9 +4,9 @@
 
 {% block content %}
 <div class="p-strip">
-  <div class="row">
-    <div class="col-6">
-      <img src="/static/confused-hex.svg" />
```

Comment:
We should remove the `confused-hex.svg` file from project if it's not used anymore.